src/main/java/frc/robot/Configs.java

```diff
@@ -4,53 +4,88 @@
+                                        .velocityConversionFactor(drivingFactor / 60.0); // meters per second
+                        drivingConfig.closedLoop
+                                        .feedbackSensor(FeedbackSensor.kPrimaryEncoder)
+                                        // These are example gains you may need to them for your own robot!
+                                        .pid(0.04, 0, 0)
+                                        .velocityFF(drivingVelocityFeedForward)
+                                        .outputRange(-1, 1);
+
+                        turningConfig
+                                        .idleMode(IdleMode.kBrake)
+                                        .smartCurrentLimit(20);
+                        turningConfig.absoluteEncoder
+                                        // Invert the turning encoder, since the output shaft rotates in the opposite
+                                        // direction of the steering motor in the MAXSwerve Module.
+                                        .inverted(true)
+                                        .positionConversionFactor(turningFactor) // radians
+                                        .velocityConversionFactor(turningFactor / 60.0); // radians per second
+                        turningConfig.closedLoop
+                                        .feedbackSensor(FeedbackSensor.kAbsoluteEncoder)
+                                        // These are example gains you may need to them for your own robot!
+                                        .pid(1, 0, 0)
+                                        .outputRange(-1, 1)
+                                        // Enable PID wrap around for the turning motor. This will allow the PID
+                                        // controller to go through 0 to get to the setpoint i.e. going from 350 degrees
+                                        // to 10 degrees will go through 0 rather than the other direction which is a
+                                        // longer route.
+                                        .positionWrappingEnabled(true)
+                                        .positionWrappingInputRange(0, turningFactor);
+                }
+        }
+
+        public static final class ArmConfigs {
+                public static final SparkMaxConfig armConfig = new SparkMaxConfig();
+
+                static {
+                        armConfig
+                                        .idleMode(IdleMode.kBrake) // Set to kBrake to hold position when not moving
+                                        .smartCurrentLimit(30); // Adjust current limit as needed
+
+                        // Configure the absolute encoder
+                        armConfig.absoluteEncoder
+                                        .inverted(false) // Set to true if the encoder reads backward
+                                        .positionConversionFactor(1.0) // Adjust based on encoder output (1.0 for
```

Comment:
This value should not be hard coded. It should be in the constants file as some "reduction" (don't name the variable "ratio" as it leaves ambiguity)